src/equations/acoustic_perturbation_2d.jl

```diff
@@ -260,10 +262,10 @@ end
     if orientation == 1
         f1 = v1_mean * v1_prime + v2_mean * v2_prime +
              c_mean^2 * p_prime_scaled / rho_mean
-        f2 = zero(eltype(u))
+        f2 = 0
```

Comment:
```suggestion
        f2 = zero(eltype(u))
```

for type stability - since we assign `f1` or `f2` to zero here and in the block below

Reply:
Thanks but I cannot get it. In both cases, `f3` has the same type as `u` so we have no need to convert here. Also, this is not an inner function.

Reply:
Fine, I got you.